morph/readability.py

```diff
@@ -0,0 +1,369 @@
+# -*- coding: utf-8 -*-
+from PyQt5.QtCore import *
+from PyQt5.QtWidgets import *
+from PyQt5.QtGui import *
+from .morphemes import MorphDb, getMorphemes
+from .morphemizer import getMorphemizerByName
+from . import readability_ui
+import csv
+import importlib
+import io
+import PyQt5
+import operator
+import os
+import re
+import sys
+
+from anki.utils import isMac
+
+from .util import dbsPath, errorMsg, infoMsg, mw, cfg1, initCfg
+
+importlib.reload( readability_ui )
+
+def atoi(text):
+    return int(text) if text.isdigit() else text
+
+def natural_keys(text):
+    '''
+    alist.sort(key=natural_keys) sorts in human order
+    http://nedbatchelder.com/blog/200712/human_sorting.html
+    (See Toothy's implementation in the comments)
+    '''
+    return [ atoi(c) for c in re.split(r'(\d+)', text) ]
+   
+class Show:
```

Comment:
Why is the class called Show (and why refer to the user's file inputs as shows) when the user might be processing any files for readability, like a book? Also, "show" is used in other variables (e.x in utils.py) in Morphman to refer to the verb as opposed to an actual tv show.

Reply:
Changed to Source.